app/scripts/modules/core/src/pipeline/config/stages/deploy/deployStage.js

```diff
@@ -9,8 +10,9 @@ const angular = require('angular');
 module.exports = angular.module('spinnaker.core.pipeline.stage.deployStage', [
   CLOUD_PROVIDER_REGISTRY,
   SERVER_GROUP_COMMAND_BUILDER_SERVICE,
+  CLUSTER_SERVICE,
 ])
-  .config(function (pipelineConfigProvider, cloudProviderRegistryProvider) {
+  .config(function ($injector, pipelineConfigProvider, cloudProviderRegistryProvider, clusterServiceProvider) {
```

Comment:
Are you using `$injector` for anything?

Reply:
Nope, that's left over from when I was mis-understanding how to get access to the cluster service in the config function; I'll remove it.